isp/api_channel_operations_for_organization.go

```diff
@@ -2256,7 +2327,7 @@ func (r ApiPostOrgSignalsRequest) GenericSignal(genericSignal []GenericSignal) A
 	return r
 }
 
-func (r ApiPostOrgSignalsRequest) Execute() (*http.Response, error) {
+func (r ApiPostOrgSignalsRequest) Execute() ([]GenericSignalResult, *http.Response, error) {
```

Comment:
We are notifying clients of the new values for the response from this endpoint?

Reply:
Yes this will go into the release notes that get generated on Github. :+1: